### ISSUE TITLE
[studio] Users should not be able to copy/paste or duplicate a page when user has no Create Content permissions

### ISSUE THREAD

**Issue by alhambrav** [craftercms/craftercms#2625]
### Expected behavior
Users should not be able to copy/paste or duplicate a page when user has no `Create Content` permissions

### Actual behavior
Users are able to copy/paste or duplicate a page when user has no `Create Content` permissions.  When you copy/paste or duplicate a page, it is essentially the same as creating new content.

### Steps to reproduce the problem
* Create a site using the website_editorial bp
* Create a user with author role
* Edit the permissions for the author role, remove the `Create Content` and `Create Folder` permissions then save your changes ( `Site Config` ->  `Configuration` -> `Permissions Mapping`)
```
<role name="author">
        <rule regex="/site/website/(?!crafter-level-descriptor\.level\.xml)(.*)">
            <allowed-permissions>
                <permission>Read</permission>
                <permission>Write</permission>
            </allowed-permissions>
        </rule>
        .
        .
        .
```
* Logout of Studio as user Admin and login as the author user
* Navigate to the articles, then right click on one of the articles.  Notice that the user cannot create content from the right click menu, but can copy/paste or duplicate an existing page
<img width="296" alt="screen shot 2018-11-14 at 6 00 27 am" src="https://user-images.githubusercontent.com/25483966/48478646-e2dbbc80-e7d2-11e8-95c5-eecd2e45f0d8.png">

### Log/stack trace (use https://gist.github.com)
Here's a short clip:
https://www.useloom.com/share/333bd871b5df4e62acc78d4275d2f7c6

#### Version
Studio Version Number: 3.1.0-SNAPSHOT-eed2f7
Build Number: eed2f7303d54ef64d7677e01f52b3e033ed40cae
Build Date/Time: 11-13-2018 22:09:07 -0500
#### OS
OS X
#### Browser
Chrome browser

**Comment by avasquez614** [comment]
@jvega190 please check if this is a backend/UI bug or if it's purely backend.

**Comment by jvega190** [comment]
https://github.com/craftercms/studio-ui/pull/986

**Comment by alhambrav** [comment]
Verified fixed.

**Comment by russdanner** [comment]
This needs to be ported to 3.0.x

**Comment by jvega190** [comment]
3.0.x: https://github.com/craftercms/studio-ui/pull/989

**Comment by alhambrav** [comment]
Verified fixed for 3.0 and 3.1
